src/Server/modules/DataClasses.js

```diff
@@ -109,6 +109,12 @@ module.exports = {
 
 			// Dictionary of { <Date, number of Recovered cases> }
 			this.numRecovered = {};
+
+			// Modifying WorldPlace to add 3 new ints for optimization
+			// Populated when num(Deaths/Confirmed/Recovered) are populated
+			this.currentNumConfirmed = 0;
```

Comment:
These variables should be populated when num(deaths/Confirmed/recovered) are populated or modified.